src/test/java/com/training/assignments/Assignment1Test.java

```diff
@@ -0,0 +1,54 @@
+package com.training.assignments;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Assignment1Test {
+    Assignment1 assign1 = new Assignment1();
+
+    @Test
+    public void intToDoubleTest()
```

Comment:
use test as prefix for method & Suffix for class. 
ex class name-->Assignment1Test 
     method name-->testIntToDouble

please do the changes accordingly for all the below methods as well.